pkg/endpoint/endpoint.go

```diff
@@ -1065,6 +1065,7 @@ func ParseEndpoint(strEp string) (*Endpoint, error) {
 	}
 
 	ep.UpdateLogger(nil)
+	ep.SecurityIdentity.Sanitize()
```

Comment:
Any possibility that the endpoint's identity is nil here?

Reply:
probably, we're reading stuff off the filesystem.

Reply:
I think in the common case there's a value here. If someone goes and messes with the FS state, they could snip this out, and I presume that would lead `parseBase64ToEndpoint()` to set the `SecurityIdentity` to nil. So then what do we do? Set it with the reserved:init identity?

Reply:
Yes

Reply:
Resolved in the latest version by setting identity to `reserved:init`.